sky/tools/create_full_ios_framework.py

```diff
@@ -163,9 +163,34 @@ def create_framework(
   ])
 
 
+def embed_codesign_configuration(config_path, contents):
+  with open(config_path, 'w') as f:
+    f.writelines(contents)
+
+
 def zip_archive(dst):
+  ios_file_with_entitlements = ['gen_snapshot_arm64\n']
+  ios_file_without_entitlements = [
+      'Flutter.xcframework/ios-arm64/Flutter.framework/Flutter\n',
```

Comment:
is the `\n` required if you are using writelines?

Reply:
yes sir

Reply:
Is it because we require a white line in between lines and at the end of the file? wouldn't this make it more difficult when reading the file?

Reply:
Yeah that's accurate, I am adding a newline in between the contents, and dart can parse it with https://api.dart.dev/stable/2.17.6/dart-convert/LineSplitter-class.html.